commit 1ba1e576e20e9008d3fd862deb7bbcca61f3e4d8
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Thu Mar 10 20:10:50 2016 +0100

    Remove duplicate KF5DocTools search, make it optional
    
    Using ecm_optional_add_subdirectory, as long as KDELibs4Support
    makes it impossible to disable KF5DocTools.
    
    REVIEW: 127335

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b0dd543..9623485 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,7 +33,6 @@ find_package(KF5 REQUIRED COMPONENTS
     NewStuff
     Plotting
 )
-find_package(KF5DocTools REQUIRED)
 find_package(Eigen3 3.2.2 REQUIRED)
 find_package(GSL)
 find_package(Qalculate)
@@ -69,7 +68,9 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-parameter")
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${EIGEN3_INCLUDE_DIR})
 
-add_subdirectory(doc)
+if(KF5DocTools_FOUND)
+    ecm_optional_add_subdirectory(doc)
+endif()
 add_subdirectory(stepcore)
 add_subdirectory(step)
 add_subdirectory(autotests)
